pkg/registry/resource/resourceclaim/strategy.go

```diff
@@ -172,4 +174,38 @@ func toSelectableFields(claim *resource.ResourceClaim) fields.Set {
 
 // dropDisabledFields removes fields which are covered by a feature gate.
 func dropDisabledFields(newClaim, oldClaim *resource.ResourceClaim) {
+	dropDisabledDRAAdminAccessFields(newClaim, oldClaim)
+}
+
+func dropDisabledDRAAdminAccessFields(newClaim, oldClaim *resource.ResourceClaim) {
+	if utilfeature.DefaultFeatureGate.Enabled(features.DRAAdminAccess) {
```

Comment:
Usually this is "if gate-enabled || alread-in-use" - is it intentional to drop the 2nd part of that?

Reply:
I am using that pattern now. Instead of trying to be (too) smart and checking if the feature is set in the old request, I'm now just checking whether it was used anywhere in the old claim.